.github/workflows/release-to-pypi.yml

```
@@ -0,0 +1,80 @@
+name: Release to PyPI
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  build-wheels:
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+
+    - name: Check out the release commit
+      uses: actions/checkout@v2
```

Could upgrade to the most recent version of the action here
```suggestion
      uses: actions/checkout@v3
```